ui/oopsyraidsy/data/05-shb/raid/e5n.js

```diff
@@ -6,9 +6,75 @@
     ko: /^희망의 낙원 에덴: 공명편 \(1\)$/,
   },
   damageWarn: {
+    '4B8F': 'Judgment Jolt', // Stratospear explosions
+    '4E3A': 'Impact', // Stratospear landing AoE
+    '4B9C': 'Lightning Bolt', // Stormcloud standard attack
+    '4B97': 'Gallop', // Sideways add charge
+    '4BA1': 'Shock Strike', // Small AoE circles during Thunderstorm
+    '4CF2': 'Volt Strike', // Large AoE circles during Thunderstorm
```

Comment:
Can you remind me if this is the same as the volt strike in e5s, where everybody gets a circle and always takes this damage?

Reply:
It is not. That one doesn't appear on normal.